api/infrastructure/deploy-api.sh

```diff
@@ -90,53 +66,29 @@ case $key in
 esac
 done
 
-if [ -z "${S3_BUCKET}" ] || [ -z "${ECR_REPO}" ] || [ -z "${AWS_DEFAULT_REGION}" ] ; then
+if [ -z "${S3_BUCKET}" ] || [ -z "${AWS_DEFAULT_REGION}" ] ; then
     echo "$usage" >&2
     exit 1
 fi
 
 ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
-ECR_ENDPOINT="${ACCOUNT_ID}.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com"
 S3_UPLOAD_URI="s3://${S3_BUCKET}/api/ParallelCluster.openapi.yaml"
-
-echo "Building docker image"
-"${SCRIPT_DIR}/../docker/awslambda/docker-build.sh"
-
-echo "Pushing docker image to ${ECR_ENDPOINT}/${ECR_REPO}"
-aws ecr get-login-password | docker login --username AWS --password-stdin "${ECR_ENDPOINT}"
-docker tag pcluster-lambda:latest "${ECR_ENDPOINT}/${ECR_REPO}:latest"
-docker push "${ECR_ENDPOINT}/${ECR_REPO}:latest"
+POLICIES_S3_URI="s3://${S3_BUCKET}/stacks/parallelcluster-policies.yaml"
+POLICIES_TEMPLATE_URI="http://${S3_BUCKET}.s3.${AWS_DEFAULT_REGION}.amazonaws.com/stacks/parallelcluster-policies.yaml"
```

Comment:
The bucket is used for uploading the file in this script while the template URI is in the format needed in the CFN template.

Reply:
Just wondering why `S3_UPLOAD_URI ` which should follow the same logic, does not need two different variables

Reply:
(the ECR_ENDPOINT is similar in that we were able to push to that repo in this script and then reference it in the template as-is because it follows the same format in the push command as where we consume it in the template)